scripts/page_browse.js

```diff
@@ -336,16 +331,15 @@ function pageBrowse(params, storageKey, replaceUrl, mentorMode = false) {
                 resolve();
             } else {
                 roundSelector = document.createElement("select");
-                $.ajax(makeApiAjaxSettings("v1/projects/pagerounds"))
-                    .done(function(rounds) {
+                ajax("GET", "v1/projects/pagerounds")
+                    .then(function(rounds) {
                         rounds.forEach(function(round) {
                             let selected = (currentRound === round);
                             roundSelector.add(new Option(round, round, selected, selected));
                         });
                         resolve();
-                    })
-                    .fail(function(jqxhr) {
-                        showError(jqxhr);
+                    }, function(error) {
```

Comment:
would it be cleaner to have a .catch instead of a 2nd param to then? I feel it is more common this way.

Reply:
That seems reasonable. Done this.